.storybook/main.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ['../stories/**/*.stories.[tj]s'],
+  stories: [STORIES],
```

Comment:
Can we add a comment/note here about the replacement? I could see being _super_ confused   a year from now 😂

Reply:
Ah okay! I added it in the doc